### ISSUE TITLE
latest 44.4.2 doesn't support falcon_mamba

### ISSUE THREAD

**Issue by thistleknot** [huggingface/transformers#33262]
### System Info

python 3.10
rocky linux 9

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

pip install transformers==4.44.2

```
Traceback (most recent call last):
  File "/home/user/miniconda3/envs/textgen/lib/python3.10/site-packages/transformers/models/auto/configuration_auto.py", line 993, in from_pretrained
    config_class = CONFIG_MAPPING[config_dict["model_type"]]
  File "/home/user/miniconda3/envs/textgen/lib/python3.10/site-packages/transformers/models/auto/configuration_auto.py", line 695, in __getitem__
    raise KeyError(key)
KeyError: 'falcon_mamba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/text-generation-webui/modules/ui_model_menu.py", line 231, in load_model_wrapper
    shared.model, shared.tokenizer = load_model(selected_model, loader)
  File "/home/user/text-generation-webui/modules/models.py", line 93, in load_model
    output = load_func_map[loader](model_name)
  File "/home/user/text-generation-webui/modules/models.py", line 155, in huggingface_loader
    config = AutoConfig.from_pretrained(path_to_model, trust_remote_code=shared.args.trust_remote_code)
  File "/home/user/miniconda3/envs/textgen/lib/python3.10/site-packages/transformers/models/auto/configuration_auto.py", line 995, in from_pretrained
    raise ValueError(
ValueError: The checkpoint you are trying to load has model type `falcon_mamba` but Transformers does not recognize this architecture. This could be because of an issue with the checkpoint, or because your version of Transformers is out of date.
```

### Expected behavior

mamba falcon loads

**Comment by vasqu** [comment]
Afair it hasn't been included in any version yet. It most likely will be released with the next major version 4.45 (so in max 1-2 weeks).

For now you could install from source. Should be something like:
`pip install git+https://github.com/huggingface/transformers.git`

**Comment by LysandreJik** [comment]
v4.45 was released, so I'm closing this (thanks, as always, @vasqu!)
